src/components/map/Map.tsx

```diff
@@ -1,64 +1,83 @@
 
-    // we need to update the key on the GEOJSON to let react know it's time to rerender
-    setForceUpdate(Math.random())
-  }, [state.explore.estimateGradePrevalences])
+        if (countryEstimate && countryEstimate.testsAdministered) {
+          const { testsAdministered, geographicalName, numberOfStudies, localEstimate, nationalEstimate, regionalEstimate, sublocalEstimate } = countryEstimate;
+          return { ...country, properties: { testsAdministered, geographicalName, numberOfStudies, localEstimate, nationalEstimate, regionalEstimate, sublocalEstimate }}
+        }
+        return { ...country, properties: { testsAdministered: null, geographicalName: null, numberOfStudies: null, localEstimate: null, nationalEstimate: null, regionalEstimate: null, sublocalEstimate: null } }
+      })
+      setMapRecords(countriesData);
+    }
+  }, [state.explore.estimateGradePrevalences, state.countries])
 
   const bounds = latLngBounds([-90, -200], [90, 180]);
   const maxBounds = latLngBounds([-90, -200], [90, 200]);
 
   const mapboxAccessToken = process.env.REACT_APP_MAPBOX_API_KEY;
+
   return (
-    <LeafletMap
-      ref={mapRef}
-      center={[0, 0]}
-      zoom={1}
+    <MapContainer
+      center={bounds.getCenter()}
+      zoom={3}
       className="map w-100"
       bounceAtZoomLimits={true}
       bounds={bounds}
-      minZoom={2}
+      minZoom={3}
       maxBounds={maxBounds}
-      enableHighAccuracy={true}
       maxBoundsViscosity={1}
     >
+
+      <CountriesTileLayer
+        url="https://tiles.arcgis.com/tiles/5T5nSi527N4F7luB/arcgis/rest/services/Countries/VectorTileServer"
+        records={mapRecords}
+        zIndex={50}
+      />
+
+      <VectorTileLayer 
+        url="https://tiles.arcgis.com/tiles/5T5nSi527N4F7luB/arcgis/rest/services/WHO_Polygon_Basemap_no_labels/VectorTileServer"
+        style={layerStyle}
+        fetchApiStyle
+      />
+
+      <VectorTileLayer 
+        url="https://tiles.arcgis.com/tiles/5T5nSi527N4F7luB/arcgis/rest/services/WHO_Polygon_Basemap_Disputed_Areas_and_Borders_VTP/VectorTileServer"
+        zIndex={70}
+        fetchApiStyle
+        style={layerStyle}
+        front
+      />
+
+      <Legend buckets={buckets} />
+
       <TileLayer
+        ref={(tl : any) => tl?.bringToFront()}
         url={getMapUrl(state.language) + mapboxAccessToken}
         attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
         tileSize={512}
-        id={'mapbox/light-v9'}
-        zoomOffset={-1}>
-      </TileLayer>
-      <GeoJSON
-        onEachFeature={(feature, layer) => onAltEachFeature(feature, layer, mapRef, state.language)}
-        ref={geoJsonRef}
-        key={forceUpdate}
-        data={mapRecords as GeoJSON.GeoJsonObject}
-        style={(data) => altStyle(data, buckets)}>
-      </GeoJSON>
-      <Legend buckets={buckets} />
-    </LeafletMap>
+        id={'serotracker/ckis9n3mt0qkg19o0q03wkg84'}
```

Comment:
Just making sure, none of these IDs and URLs contain anything that should be kept secret right?

Reply:
I think this is a key on this line

Reply:
This isn't the mapbox token, its an id for the labels mapbox layer we created, which is public. this is the pattern that was used before my update so I think its okay